### ISSUE TITLE
Learning rate for LSUV vs. Batch Norm

### ISSUE THREAD

**Issue by bhokaal2k** [ducha-aiki/LSUVinit#1]
Hi,
    Excellent work and code !!
I have a question related to the experimental comparison between LSUV initialized model and BN model training progress. Did you use the same learning rate for both the approaches ? If not, could you please shed some light on how the learning rate for the LSUV init model was adjusted. Also, what would be your general recommendation in terms of the LR of LSUV init wrt BatchNorm model. Does any particular optimization seems to clearly favor LSUV init i.e. SGD, Nesterov, AdaGrad, Adam etc.

Thanks for your time and effort.

**Comment by ducha-aiki** [comment]
Hi,

Yes, I have used same learning rate for both lr= 0.01. It seems, it is maximum you can get good results on CIFAR. It was adjusted very simple way: 0.01 is kind of default setup and works well for lots of datasets and models. We have also tried 0.1, 0.05, 0.02 and 0.005. They give same or worse performance.

As for BN - model, the LSUV init does not help much for it, because all normalization is done inside batchnorm. Except, may be orthonormal weights initialization - explicit orthogonalization is better, than approximate one from random gaussian.
Unfortunately, seems that LSUV init could stand vs BatchNorm only on small datasets. The benchmark on ImageNet I have started after finishing the paper shows it https://github.com/ducha-aiki/caffenet-benchmark

Optimization. The best results on CIFAR10 were obtained with RMSProp. SGD and Nesterov are also good, haven`t tried any adaptive solvers.

**Comment by bhokaal2k** [comment]
Hi,
    Thanks a lot for the details and the detailed comparison link.

Thanks a tonne.
